app/mailers/bounty_mailer.rb

```diff
@@ -0,0 +1,7 @@
+class BountyMailer < ApplicationMailer
+  def open_bounty
+    @user = params[:user]
+    @bounties = params[:bounties]
+    mail(to: @user.email, subject: "Here are the latest open bounties")
+  end
+end
```

Comment:
Mailers have been updated to be more like Controllers and now accept a params hash to pass you information to the mailer 

https://guides.rubyonrails.org/action_mailer_basics.html#mailers-are-similar-to-controllers


We also don't need the default `from` or `to` values here, most of the time, the default values belong in the `ApplicationMailer`.

While it's a good idea to pass all the emails in at once, typically, you want your email to have only 1 recipient.  You had a great idea with that and using `bcc` was a good call to not expose all the email addresses to the users.

We also want to keep things flexible if it ever fails.  By updating the email to take only 1 email address, if it fails we can just try again for that single user instead of adding in all the user email's and potentially sending multiple emails to the same user.